scripts/seed-local-env.ts

```diff
@@ -20,6 +20,7 @@ const usageReportingEndpoint =
     : envName === 'dev'
       ? 'https://app.dev.graphql-hive.com/usage'
       : 'http://localhost:4001';
+const target = process.env.TARGET;
```

Comment:
I did this for testing because I was using a new access token and they require passing in the target.